test/action_controller/json/include_test.rb

```diff
@@ -36,6 +36,11 @@ def render_without_include
             render json: @author, adapter: :json
           end
 
+          def render_without_associations
+            setup_data
+            render json: @author, adapter: :json, include: []
```

Comment:
unless you're testing the controller, unit test is better :)  also, see test/support, there's some serialization helpers in there